submissions/AnthoniaAdollo/aboutMe/AboutME.rtf

```diff
@@ -0,0 +1,11 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2509
```

Comment:
@AAOYV2020 You need to delete this file from your pull request. It is not part of the success criteria.

Reply:
I'm a tad confused. This is the 'aboutMe' folder I created within the name folder. I need some help with navigating this, please.